contessa/models.py

```diff
@@ -136,7 +134,37 @@ def __repr__(self):
         return f"Rule ({self.attribute} - {self.rule_name} - {self.task_ts})"
 
 
-def get_default_qc_class(table):
+# todo - maybe create also CheckTable
+class Table:
+    def __init__(self, schema_name, table_name):
+        self.schema_name = schema_name
+        self.table_name = table_name
+
+    @property
+    def fullname(self):
+        # todo - solve sanitization. problem - can be already sanitized
+        return f"{self.schema_name}.{self.table_name}"
+
+
+class QualityTable(Table):
```

Comment:
Name is too short to be descriptive. E. g. later in the code it is used as `result_table`. Also its parameter `quality_name_` can be `full_name` or `quality_check_(table_)name`.

Reply:
changed to ResultTable